src/components/autocompleteInput.js

```diff
@@ -622,6 +623,36 @@
 
     const currentValue = getValue();
 
+    useEffect(() => {
+      let debounceCurrentValue;
+
+      if (!isListProperty) {
+        if (currentValue !== debouncedCurrentValue) {
+          debounceCurrentValue = setTimeout(() => {
+            setDebouncedCurrentValue(currentValue);
+          }, 250);
+        } else {
+          let triggerEventValue;
+
+          if (!isListProperty) {
```

Comment:
you already checked what `!isListProperty` is, right?

Reply:
I have removed the first check since we want to use the onchange interaction with the list property and relation property